packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketVerifyAccount.tsx

```diff
@@ -75,13 +75,25 @@ const getSuiteReceiveAccounts = ({
                 !unavailableCapabilities[n.symbol] &&
                 ((n.isDebugOnlyNetwork && isDebug) || !n.isDebugOnlyNetwork),
         );
+
+        const isSameDevice = (a: Account) => a.deviceState === device?.state;
+        const isSameNetwork = (a: Account) => a.symbol === receiveNetwork;
+        const isDebugAndIsAccountDebugOnly = (a: Account) =>
+            isDebugOnlyAccountType(a.accountType, a.symbol) && isDebug;
+        const isNotDebugOnlyAccount = (a: Account) =>
+            !isDebugOnlyAccountType(a.accountType, a.symbol);
+        const isNotEmpty = (a: Account) => !a.empty;
+        const isVisible = (a: Account) => a.visible;
+        const isFirstNormalAccount = (a: Account) => a.accountType === 'normal' && a.index === 0;
+
         if (receiveNetworks.length > 0) {
             // get accounts of the current symbol belonging to the current device
```

Comment:
you can move all your helper functions here.

use `account` instead of `a` 

(code style guide 😡 )